src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,19 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTab, setActiveTab }) => (
```

Comment:
<img width="634" alt="Screenshot 2024-01-03 at 15 39 25" src="https://github.com/mate-academy/react_tabs-js/assets/135232001/1e11afae-9183-49fd-84c3-a4cfe291b195">
The default tab should be defined here if there is an issue with the active tab prop